scripts/backup.sh

```diff
@@ -17,6 +17,6 @@ fi
 echo "backup created at $FILE_PATH"
 
 if [ "${DELETE_OLD_BACKUPS}" = true ]; then
-        echo "removing backups older than ${OLD_BACKUP_DAYS:=30} days"
-        find /palworld/backups/ -mindepth 1 -maxdepth 1 -mtime "+${OLD_BACKUP_DAYS}" -print -delete
+        echo "removing backups older than ${OLD_BACKUP_DAYS} days"
```

Comment:
Removing the default 30 days could result in some errors if the OLD_BACKUP_DAYS isn't set

Reply:
I moved OLD_BACKUP_DAYS to the dockerfile so the default it defined in there. Added if statements to verify it is not empty and is a number as to avoid other errors.